client/src/pages/Kanban.vue

```diff
@@ -27,91 +27,120 @@
 </template>
 
 <script>
-import { ref, onMounted } from 'vue'
-import draggable from 'vuedraggable'
-import RedmineService from '@/services/RedmineService.js'
-import { useStore } from "vuex"
+  import { ref, onMounted } from 'vue'
+  import draggable from 'vuedraggable'
+  import RedmineService from '@/services/RedmineService.js'
+  import { useStore } from "vuex"
+  import lodash from "lodash"
 
-export default {
-  name: "Kanban",
-  components: {
-    draggable
-  },
-  setup() {
-    let uniqueStatusNamesWithIds
-    let originalIssues
-    const store = useStore()
-    let issuesByStatus = ref([])
+  export default {
+    name: "Kanban",
+    components: {
+      draggable
+    },
+    setup() {
+      const issueIdRegex = /\d+/
+      const store = useStore()
+      const columnConfig = ref([])
+      const wipLimit = ref()
+      const fallbackColumnConfig = ["Új", "Folyamatban", "Megoldva"]
+      const fallbackWipLimit = 20
 
-    function log() {
-      //window.console.log(evt)
-    }
+      let issuesForProject = []
+      let issuesByStatus = ref()
 
-    async function getIssuesForProject(){
-      let response = (await RedmineService.getIssuesForProject(store.state.user.api_key, store.state.query.id, store.state.project.id)).data
-      originalIssues = response.issues
+      function log() {
+        //window.console.log(evt)
+      }
 
-      const uniqueStatusNames = response.issues.reduce((acc, curr) => {
-        return acc.includes(curr.status.name) ? acc : [...acc, curr.status.name]
-      }, []);
+      async function setupColumnConfig() {
+        let redmineStatuses
+        let configIssue
+        let columnNames
+        let wipLimitFromConfig
+        try {
+          redmineStatuses = (await RedmineService.getRedmineStatuses(store.state.user.api_key)).data.issue_statuses
+          configIssue = (await RedmineService.getKanbanConfig(store.state.user.api_key, store.state.project.id)).data.issues[0]
+          let config = JSON.parse(configIssue.description).config
+          columnNames = config.columns || fallbackColumnConfig
+          wipLimitFromConfig = config.WIP || fallbackWipLimit
```

Comment:
seems a bit redundant since it's the same as the catch, what is the reason behind it?